pyboy/plugins/window_sdl2.py

```diff
@@ -181,7 +184,14 @@ def post_tick(self):
         sdl2.SDL_RenderClear(self._sdlrenderer)
 
     def enabled(self):
-        return self.pyboy_argv.get("window_type") == "SDL2" or self.pyboy_argv.get("window_type") is None
+        if self.pyboy_argv.get("window_type") in ("SDL2", None):
+            if not sdl2:
+                logger.error("Failed to import sdl2, needed for sdl2 window")
+                return False  # Disable, or raise exception?
+            else:
+                return True
+        else:
+            return False
```

Comment:
Same as before:
```python
if not sdl2:
    logger.error("Failed to import sdl2, needed for SDL2 window")
    return False
return self.pyboy_argv.get("window_type") == "SDL2" or self.pyboy_argv.get("window_type") is None
```
Exceptions are tricky in Cython.

Reply:
```suggestion
        if self.pyboy_argv.get("window_type") in ("SDL2", None):
            if sdl2:
                return True
            else:
                logger.error("Failed to import sdl2, needed for sdl2 window")
            return False
```

Not sure there's a shorter way than that if we want it to log the error correctly